### PR TITLE
applications: nrf5340_audio: Use PD from BASE

### PR DESCRIPTION
**Pull request by alexsven**
- Use presentation delay from BASE
- Clean up active_stream when cleaning up BIS

OCT-2714

### CODE REVIEW

**Comment by alexsven** [comment]
Tested locally and verified that PD is still 10ms after this change

**Review by rick1082 — APPROVED** [review]
Great to see we can use PD from BASE!

File: applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c [hunk 1]
```diff
@@ -44,6 +44,7 @@ struct audio_codec_info {
 struct active_audio_stream {
 	struct bt_bap_stream *stream;
 	struct audio_codec_info *codec;
+	uint32_t pd;
```
Comment (koffes) [inline applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c:47]: Suggest we rename to pres_dly or pres_delay

File: applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c [hunk 2]
```diff
@@ -449,7 +455,7 @@ int broadcast_sink_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32
 			return -ENXIO;
 		}
 
-		*pres_delay = active_stream.stream->qos->pd;
+		*pres_delay = active_stream.pd;
```
Comment (koffes) [inline applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c:458]: @Thalley : Can you have a quick eye on this?

**Review by koffes — APPROVED** [review]

**Comment by koffes** [comment]
Waiting for approval from @Thalley.

File: applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c (hunk 1 above)
Comment (alexsven) [reply applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c:47]: The struct member in base is called pd, that's why I chose the same name for this.

**Review by Thalley — APPROVED** [review]

**Comment by cvinayak** [comment]
Do you create issues for bugfixes, that you can refer to in the describe of this PR? (asking to make sure I may be merging without bug reports)...

**Comment by alexsven** [comment]
> Do you create issues for bugfixes, that you can refer to in the describe of this PR? (asking to make sure I may be merging without bug reports)...

Yes, the issue number is in the branch name

**Pull request merged**